spot-electron/package.json

```diff
@@ -26,15 +27,17 @@
         "dist": "electron-builder",
         "lint": "eslint .",
         "pack": "electron-builder --dir",
+        "postinstall": "electron-rebuild",
         "start": "electron ."
     },
     "version": "1.0.0",
     "dependencies": {
         "jitsi-meet-logger": "github:jitsi/jitsi-meet-logger#5da3033d647054457597920252b3387fa43a17fa",
         "lodash": "4.17.11",
-        "node-osascript": "^2.1.0"
+        "node-osascript": "2.1.0"
     },
     "optionalDependencies": {
-        "win-audio": "2.0.2"
+        "win-audio": "2.0.2",
+        "xpc-connect": "git+https://git@github.com/jitsi/xpc-connect.git#33fd79673c57f15167067778fc6ade0a4f46da07"
```

Comment:
Should we wait a little bit to see what the response is from the PR?

Reply:
Not sure, because I also changed the package.json, that I think they don't want to merge

Reply:
Looks like the maintainer is okay with them. Let's wait a bit if that's okay. I'm assuming "a bit" is less than a couple days.

Reply:
Yeah, had to rebase now. We'll see